.ort.yml

```diff
@@ -5,6 +5,10 @@ excludes:
     reason: "TEST_OF"
     comment: >-
       Licenses contained in this directory are used for testing and do not apply to the OSS Review Toolkit.
+  - pattern: "docs/**"
+    reason: "DOCUMENTATION_OF"
+    comment: >-
+      This directory contains project website with examples using licenses that do not apply to the OSS Review Toolkit.
```

Comment:
The "examples [...] that do not apply to the OSS Review Toolkit" part sounds a bit odd to me, as if we were giving examples that are not workable.

Maybe better something like: "This directory contains documentation that is not distributed as part ORT releases."